generated/runtime.json

```diff
@@ -1536,6 +1536,1710 @@
+            "vargaz"
+          ]
+        },
+        {
+          "labels": [
+            "area-CodeGen-coreclr"
+          ],
+          "mentionees": [
+            "dotnet/jit-contrib",
+            "AndyAyersMS",
+            "BruceForstall",
+            "EgorBo",
+            "erozenfeld",
+            "jakobbotsch",
+            "JulieLeeMSFT",
+            "kunalspathak",
+            "markples",
+            "SingleAccretion",
+            "tannergooding",
+            "TIHan"
+          ]
+        },
+        {
+          "labels": [
+            "area-Codegen-Interpreter-mono"
+          ],
+          "mentionees": [
+            "BrzVlad",
+            "kotlarmilos"
+          ]
+        },
+        {
+          "labels": [
+            "area-Codegen-Intrinsics-mono"
+          ],
+          "mentionees": [
+            "fanyang-mono"
+          ]
+        },
+        {
+          "labels": [
+            "area-Codegen-JIT-mono"
+          ],
+          "mentionees": [
+            "vargaz"
+          ]
+        },
+        {
+          "labels": [
+            "area-Codegen-Jiterpreter-mono"
+          ],
+          "mentionees": [
+            "kg"
+          ]
+        },
+        {
+          "labels": [
+            "area-Codegen-LLVM-mono"
+          ],
+          "mentionees": [
+            "vargaz"
+          ]
+        },
+        {
+          "labels": [
+            "area-Codegen-meta-mono"
+          ],
+          "mentionees": [
+            "vargaz"
+          ]
+        },
+        {
+          "labels": [
+            "area-CoreLib-mono"
+          ],
+          "mentionees": [
+            "vargaz"
+          ]
+        },
+        {
+          "labels": [
+            "area-CrossGen/NGEN-coreclr"
+          ],
+          "mentionees": [
+            "dotnet/crossgen-contrib",
+            "cshung",
+            "davidwrighton",
+            "ivdiazsa",
+            "jakobbotsch",
+            "janvorli",
+            "jkotas",
+            "mangod9",
+            "MichalStrehovsky",
+            "trylek"
+          ]
+        },
+        {
+          "labels": [
+            "area-crossgen2-coreclr"
+          ],
+          "mentionees": [
+            "trylek",
+            "dotnet/crossgen-contrib",
+            "cshung",
+            "davidwrighton",
+            "ivdiazsa",
+            "jakobbotsch",
+            "janvorli",
+            "jkotas",
+            "mangod9",
+            "MichalStrehovsky"
+          ]
+        },
+        {
+          "labels": [
+            "area-Debugger-mono"
+          ],
+          "mentionees": [
+            "thaystg",
+            "radical"
+          ]
+        },
+        {
+          "labels": [
+            "area-DependencyModel"
+          ],
+          "mentionees": [
+            "dotnet/area-dependencymodel",
+            "carlossanlop",
+            "ericstj",
+            "jeffhandley",
+            "smasher164",
+            "ViktorHofer"
```

Comment:
This approach is currently producing a material difference in how `mentionees` are populated: The `@dotnet/` teams are expanded and instead of only mentioning the team, it's mentioning the team and all members of the team.

This is caused due to a difference in the data we included in `area-owners.md` and `area-owners.json`, where `area-owners.md` stated the mentionees (teams + external contributors who want to be subscribed), and `area-owners.json` expands those teams for sake of the https://issuesof.net queries allowing `area-owner:{team-member}`.